pop-api/examples/fungibles/README.md

```diff
@@ -0,0 +1,41 @@
+# PSP22 Fungible Token with Pop API
+
+
+This [ink!][ink] contract shows a contract that allows interaction and management of a fungible token following the [PSP22 standard][psp22], utilizing the [Pop API Fungibles][pop-api-fungibles] feature. In this specific contract only the contract owner has a permission to call specific methods.
+
+## Design Goals
+
+- Token exists as an asset on the Pop Network, rather than being confined solely to the contract.
+- Contract is the `origin` of the calls made by `pop-api`.
+- Only the contract owner has a permission to call specific methods.
+
+[Learn more how Pop API works.](/pop-api/README.md)
```

Comment:
Remove this paragraph, design goals is not really suitable for this readme. Have we designed anything you think :) ?